parts/inc/uv

```diff
@@ -495,7 +512,7 @@ else {
         {                 # Old algorithm supposedly failed to detect this
             input      => "\xff\x80\x90\x90\x90\xbf\xbf\xbf\xbf\xbf\xbf\xbf\xbf",
             adjustment => 0,
-            warning    => qr/overflow/,
+            warning    => $] <= 5.008006 ? qr/\QMalformed UTF-8 character\E/ : qr/overflow/,
```

Comment:
No need for `\Q` and `\E` as there is no interpolation.

```suggestion
            warning    => $] <= 5.008006 ? qr/Malformed UTF-8 character/ : qr/overflow/,
```

Reply:
fine with it, thanks